x-pack/legacy/plugins/uptime/public/components/functional/filter_group/index.ts

```diff
@@ -4,4 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export { FilterGroup } from './filter_group';
+import { connect } from 'react-redux';
+import { setFilters } from '../../../state/actions';
+import { FilterGroup as FilterGroupComponent } from './filter_group';
+
+const mapDispatchToProps = (dispatch: any) => ({
+  setFilters: (filters: Map<string, string[]>) => dispatch(setFilters(filters)),
+});
+
+export const FilterGroup = connect(null, mapDispatchToProps)(FilterGroupComponent);
```

Comment:
I do have a bit of a concern about edits to this component and associated files - this part of the app has heavy modifications in https://github.com/elastic/kibana/pull/50837, it may be best to wait until it's merged and we can resolve any redundancies.

Reply:
Sure, we can wait for those PR.